File: ossDotNetApp/Models/ServiceCost.cs
using System;
using System.Collections.Generic;

namespace ossDotNetApp.Models
{
    public partial class ServiceCost
    {
        public int ServiceCostid { get; set; }
        public decimal Cost { get; set; }
        public int ServiceId { get; set; }
        public int VendorId { get; set; }

        public virtual Service Service { get; set; } = null!;
        public virtual Vendor Vendor { get; set; } = null!;
    }
}

File: ossSpringApp/src/main/java/com/example/demo/entities/ServiceCost.java
package com.example.demo.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name="service_cost")
public class ServiceCost {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int service_costid;
	
	double cost;
	
	@ManyToOne
	@JoinColumn(name="service_id")
	Services service_id;
	
	@ManyToOne
	@JoinColumn(name="vendor_id")
	Vendor vendor_id;
}
